backend/src/Command.ts

```diff
@@ -34,6 +33,15 @@ export default class Command {
     if (!this.quiet) Logger[type](message, params)
   }
 
+  parseCliErrors(stderr: string) {
+    const jsonError = stderr.substring(0, stderr.indexOf('}') + 1)
+    if (jsonError) {
+      const { details }: CliStderr = JSON.parse(jsonError)
```

Comment:
Is this because the stderr may be empty?

Reply:
yes, and also to handle if the stderr isn't returned as json.